src/libraries/System.Runtime/tests/System.IO.FileSystem.Tests/DirectoryInfo/CreateSubdirectory.cs

```diff
@@ -235,6 +236,117 @@ public void ParentDirectoryNameAsPrefixShouldThrow()
             Assert.Throws<ArgumentException>(() => di.CreateSubdirectory(Path.Combine("..", randomName + "abc", GetTestFileName())));
         }
 
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsWindows), nameof(PlatformDetection.IsSubstAvailable))]
+        public void CreateSubdirectoryFromRootDirectory_Windows()
+        {
+#if TARGET_WINDOWS
+            // On Windows, use VirtualDriveHelper to create a test root directory
+            using VirtualDriveHelper virtualDrive = new();
+            char driveLetter = virtualDrive.VirtualDriveLetter;
+            string rootPath = $"{driveLetter}:\\";
+            
+            DirectoryInfo rootDir = new DirectoryInfo(rootPath);
+            string subDirName = GetTestFileName();
+            
+            // This should work without throwing ArgumentException
+            DirectoryInfo result = rootDir.CreateSubdirectory(subDirName);
+            
+            Assert.NotNull(result);
+            Assert.Equal(Path.Combine(rootPath, subDirName), result.FullName);
+            Assert.True(result.Exists);
+            
+            // Clean up
+            result.Delete();
+#endif
+        }
+
+        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        public void CreateSubdirectoryFromRootDirectory_Unix()
+        {
+            // Skip if not on Unix or not privileged
+            if (PlatformDetection.IsWindows || !PlatformDetection.IsPrivilegedProcess)
+                return;
```

Comment:
Use `[PlatformSpecific(TestPlatforms.Windows)]`.

Reply:
Move `PlatformDetection.IsPrivilegedProcess` to ConditionalFact, you need to combine multiple conditions, add a property in this class combining them.